### DISCUSSION TITLE
Organization level FUNDING.yml not showing on my organization repos

### DISCUSSION THREAD

**Discussion by onedr0p** [community/community#43045] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi 👋🏼 

I cannot get the org level `FUNDING.yml` working on any of my repositories.

Org/Personal level configuration file: https://github.com/onedr0p/onedr0p/blob/main/.github/FUNDING.yml

Sponsor button not showing up here, or any of my repos, e.g...

- https://github.com/onedr0p/home-ops

In the repo settings I only see this option:

<img width="766" alt="image" src="https://user-images.githubusercontent.com/213795/210174214-062b46fa-ff46-4ed8-b8ce-8e56dac80929.png">

Any help would be great.

**Comment by cheshire137** [comment]
Hi there! I'm a dev on the Sponsors team. I just checked your home-ops repo and I do see the Sponsor button:

![Sponsor button shown at top of home-ops repo page](https://user-images.githubusercontent.com/82317/211073790-e2e892e6-c89d-4fb4-9ee5-954c044d897f.png)

----

![Sponsor button shown in the right sidebar of home-ops repo](https://user-images.githubusercontent.com/82317/211073841-7f78e1d4-f46c-42bc-887e-e3c1480399a2.png)

**Reply by onedr0p** [reply]
Hi @cheshire137 This is because I added the [FUNDING.yml ](https://github.com/onedr0p/home-ops/blob/main/.github/FUNDING.yml)to the repo in question, for example, the following repo **does not** have a `FUNDING.yml` and the sponsor does not show up.

https://github.com/onedr0p/exportarr

<img width="799" alt="image" src="https://user-images.githubusercontent.com/213795/211074653-d60f1cca-1cb1-4e83-9d60-5e2538e5e7e7.png">

**Reply by cheshire137** [reply]
Ahh gotcha, do you have another repo where you haven't added a repo-specific funding file and you were expecting a Sponsor button to appear based on you having a user-level funding file?

**Reply by onedr0p** [reply]
Here's a few, but it's pretty much all my repos with the exception of two where I've added repo level `FUNDING.yml`
- https://github.com/onedr0p/exportarr
- https://github.com/onedr0p/containers
- https://github.com/onedr0p/sops-pre-commit

**Reply by cheshire137** [reply, marked as answer]
Ahh I think you put the user-level funding file in the wrong place: you need it in a repository called `.github`, like https://github.com/babel/.github/blob/master/FUNDING.yml. See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#creating-a-repository-for-default-files.

**Reply by onedr0p** [reply]
It is though...

```https://github.com/onedr0p/onedr0p/blob/main/.github/FUNDING.yml```

**Reply by cheshire137** [reply]
No no, you're in a repo called `onedr0p` in a directory called `.github`. You need it to be at the root of this (currently nonexistent) repository: https://github.com/onedr0p/.github

**Reply by onedr0p** [reply]
omg 🤦🏼 that was it thank you!

**Reply by cheshire137** [reply]
Haha, no problem! My eyes definitely glazed over repo-versus-folder seeing ".github" in there.
